src/base/static/components/organisms/dashboard/chart-wrapper.tsx

```diff
@@ -0,0 +1,229 @@
+/** @jsx jsx */
+import * as React from "react";
+import { jsx, css } from "@emotion/core";
+import PropTypes from "prop-types";
+
+import { SmallText, TinyTitle } from "../../atoms/typography";
+import { RadioInput } from "../../atoms/input";
+import { FontAwesomeIcon } from "../../atoms/imagery";
+import { FreeDonutChart, getFreeDonutChartData } from "./free-donut-chart";
+import { FreeBarChart, getFreeBarChartData } from "./free-bar-chart";
+import { MapseedLineChart, getLineChartData } from "./line-chart";
+import { StatSummary, getStatSummaryData } from "./stat-summary";
+import { FixedTable, getFixedTableData } from "./fixed-table";
+import { FreeTable, getFreeTableData } from "./free-table";
+import { placePropType } from "../../../state/ducks/places";
+
+const chartWrapperPropTypes = {
+  widget: PropTypes.shape({
```

Comment:
This type should live in our Dashboard duck, ideally as a TS type saved under `dashboard.d.ts` alongside the js file.

It will also allow us to re-use this type throughout the codebase, like in our function definitions, etc.